proto/src/examples/dummy_client.rs

```diff
@@ -54,66 +56,3 @@ impl Client {
         self.client.queue_test(test_request).await
     }
 }
-
-#[cfg(test)]
```

Comment:
Moved these tests to `proto/src/examples/mod.rs` because they use both the dummy server and dummy client